public/src/components/PrimaryButton.js

```diff
@@ -1,25 +1,21 @@
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 
 export const PrimaryButton = styled.button`
-  padding: .25em 1em;
-  border-radius: .25em;
-  border: 2px solid #586994;
-  width: 100%;
-  vertical-align: middle;
-  font-weight: bold;
-  font-size: 1em;
-  cursor: pointer;
-  margin-top: 1em;
+  ${({ theme }) => css`
+    ${theme.text.contentText};
+    ${theme.button.defaultStyling}
+    &:focus {
+      outline: 0;
+      box-shadow: ${theme.button.boxShadowSizes} #a9c1fc;
+    }
+  `}
+  border-color: #586994;
```

Comment:
colors are definitely something that should be in the theme

Reply:
Fixed!